install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -422,20 +422,24 @@ data:
   enable-ip-masq-agent: "true"
 {{- end }}
 
-{{- if hasKey .Values "wireguard" }}
-{{- if .Values.wireguard.enabled }}
-  enable-wireguard: {{ .Values.wireguard.enabled | quote }}
+{{- if .Values.encryption.wireguard.enabled }}
+  enable-wireguard: {{ .Values.encryption.wireguard.enabled | quote }}
+{{- if hasKey .Values.encryption.wireguard "wireguardsubnetv4" }}
+  wireguard-subnet-v4: {{ .Values.encryption.wireguard.wireguardsubnetv4 | quote }}
+{{- end }}
+{{- if hasKey .Values.encryption.wireguard "wireguardsubnetv6" }}
+  wireguard-subnet-v6: {{ .Values.encryption.wireguard.wireguardsubnetv6 | quote }}
```

Comment:
Note: The subnet options will be removed again in an upcoming PR (#15547), so maybe it makes sense to not add them there.

Reply:
Perfect, wasn't sure about this point 👍🏻